controllers/partner.controller.js

```
@@ -0,0 +1,424 @@
+        city,
+        region,
+        website,
+        mainImageDescription,
+        secondaryImageDescription,
+      } = req.body;
+      const isLocalNews =
+        req.body.isLocalNews === "true"
+          ? true
+          : req.body.isLocalNews === "false"
+          ? false
+          : partner.isLocalNews;
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)